### PR TITLE
IA-1840: new variable alb_is_internal was added

### PR DESCRIPTION
**Pull request by jsmigovsky**

### CODE REVIEW

File: alb.tf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 resource "aws_lb" "alb" {
   name               = "${var.family}-alb"
-  internal           = false
+  internal           = var.alb_is_internal
   load_balancer_type = "application"
   security_groups    = [aws_security_group.alb.id]
   subnets            = length(var.public_subnet_ids) > 0 ? var.public_subnet_ids : data.aws_subnet_ids.default[0].ids
```
Comment (bushong1) [inline alb.tf:6]: What if you made internal vs external conditional on if a public subnet was provided?

Comment (jsmigovsky) [reply alb.tf]: Just made an amend commit, see if you like this a little better.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v3.2.2
+
+* If only `private_subnet_ids` are provided then the Fargate service and the ALB will be placed in them and will become an internal-facing load-balancer.
+* If only `public_subnet_ids` are provided then the Fargate sercice and the ALB will be placed in them and will become a public-facing load-balancer.
+* If both `private_subnet_ids` and `public_subnet_ids` are NOT provided then the default VPC subnets will be assumed and will become a public-facing load-balancer.
```
Comment (bushong1) [inline CHANGELOG.md:7]: Style suggestion: swap the "double-negative" line with the "double-positive" one below to let the affirmative go first, and rephrase to ```If neither  `private_subnet_ids` nor `public_subnet_ids` are provided...```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v3.2.2
+
+* If only `private_subnet_ids` are provided then the Fargate service and the ALB will be placed in them and will become an internal-facing load-balancer.
+* If only `public_subnet_ids` are provided then the Fargate sercice and the ALB will be placed in them and will become a public-facing load-balancer.
```
Comment (bushong1) [inline CHANGELOG.md:6]: `service`

**Review by bushong1 — APPROVED** [review]

**Pull request merged**